### ISSUE TITLE
PS5 Users

### ISSUE THREAD

**Issue by mmf53** [Pugmatt/BedrockConnect#463]
### What happened?

Since server 173.82.100.84 closed, all the ps5 users in my server cannot connect. 104.238.130.180 is not working for PS5, so we always used 173.82.100.84. 207.244.229.200 seems to not work either for PS5. Anything we can do to get playstation to connect?

### Expected Behaviour?

PS5 connect to bedrockconnect.

### Steps to reproduce.

Settings/Network/Settings/Set Up Internet Connection

### Console

Playstation 5

**Comment by JefferyPlayz** [comment]
I run a bedrock server and all of my players use DNS to connect to third party servers. This problem is happening to all of my users. In the mean time, as an alternitive i've gotten them all to use the BedrockTogether app instead. Please fix this issue! 

It also seems it has happened to switch users as well, the 104.238.130.180 DNS is giving the error 'Host is using an older version of minecraft'.

**Comment by JefferyPlayz** [comment]
It doesn't only seem to be PS5 but also PS4.

**Comment by mmf53** [comment]
Yes it is happening on ps4 as well. the other issue 'Host is using an older version of minecraft', is a different issue. But the playstation one is a pain

**Comment by JefferyPlayz** [comment]
@Pugmatt 

Great news! It worked for my friends on PS5! This probably means it would work on all Playstations.

**Comment by Pugmatt** [comment]
Sweet! I'd recommend for now that PS4/PS5 players use that DNS server IP 45.55.68.52. I've added the IP address to the README under available instances

Due to the nature of the DNS server configuration changes here I'm going to leave the main instance, 104.238.130.180, without these changes for now. I've unfortunately have had issues with my host for 104.238.130.180 when trying to set a similar DNS server configuration, and generally these changes will lead to quite a bit heavier load on the server which could be an issue on the main instance. So 45.55.68.52 will act in place here for now as an alternative for PS4/PS5 similar to 173.82.100.84 was.

**Comment by mmf53** [comment]
Thank you!!! It works for Playstation again. Appreciate it.

**Comment by Pugmatt** [comment]
Nice! Thank you for the confirmations. Closing this for now unless further issues arise.

**Comment by jedipunkz** [comment]
@Pugmatt Thanks great work. I was able to work around the issue by using 45.55.68.52, but since yesterday the server list popup has stopped appearing again.

**Comment by Pugmatt** [comment]
> @Pugmatt Thanks great work. I was able to work around the issue by using 45.55.68.52, but since yesterday the server list popup has stopped appearing again.

I would try running through the [troubleshooting](https://github.com/Pugmatt/BedrockConnect/wiki/Troubleshooting#joining-the-featured-servers-still-directs-to-the-actual-servers-and-not-the-serverlist) under "Joining the Featured Servers still directs to the actual servers and not the serverlist..." if you haven't already, as the DNS appears to be working still for other PS4/PS5 users so it may be related to a different issue.
